include/deal.II/lac/block_vector_base.h

```diff
@@ -2264,6 +2262,17 @@ void BlockVectorBase<VectorType>::extract_subvector_to (ForwardIterator
     }
 }
 
+
+template <typename VectorType>
+inline
+bool BlockVectorBase<VectorType>::supports_distributed_data() const
+{
+  // Create a dummy vector of type VectorType to check
+  // supports_distributed_data even if BlockVectorBase contains no block
+  VectorType dummy;
+  return dummy.supports_distributed_data();
```

Comment:
@tamiko the property is type bound, it only depends on the template ```VectorType```. That's why, right now, it is a static variable...
 - I don't see how a static member function will work. Unless we decide that the function is not part of  ```VectorSpaceVector``` which will necessitate to cast down the derived ```Vector``` all the time. Moreover, if in the future someone wants to create a distributed vector which derives from ```ReadWriteVector```, it won't work. 
 - isn't type trait C++11?